src/containers/Home.js

```diff
@@ -85,6 +86,9 @@ const Container = styled.div`
   width: 100%;
   margin: 0 auto;
   background-color: white;
+  ${mq.mobileOnly`
+    overflow: hidden;
```

Comment:
What if we make it `overflow-x: hidden`?

Reply:
In my experiment, it also breaks `position: sticky;` on desktop. (tested on Chrome, Firefox)

Reply:
Ya, my fault. Since overflow/overflow-x/overflow-y: hidden will always try to create a new scroll container.  It might not be possible to make the sticky attribute stick to document.